### PR TITLE
add the reusable footer part

### PR DESCRIPTION
**Pull request by lario0913**
screen
mobile

### CODE REVIEW

File: dae-alright/src/components/Footer/Footer.scss [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+    .deyAlrightFooterRow-1, .deyAlrightFooterRow-2{
+        @include footer-flex;
+        @include footer-media-query(600px){
+           flex-direction: column;
+        }
+
+
+        
+        .row1col1, .row2col1{
```
Comment (SEUNAGBEYE) [inline dae-alright/src/components/Footer/Footer.scss]: Good job so far, I love what I'm seeing. But I think you can do better with your `class names`. When I see a CSS `classname`, I want to know what it's for just by looking at the name. Unfortunately, `row2col1`, `row1col2` all don't have that effect.

**Review by SEUNAGBEYE — CHANGES_REQUESTED** [review]

File: dae-alright/src/components/Footer/Footer.scss (hunk 1 above)
Comment (lario0913) [reply dae-alright/src/components/Footer/Footer.scss]: class names have been adjusted to reflect what they do..... here is the commit link
https://github.com/devcareer/Dae-Alright-Frontend/pull/4/commits/67e309ab69263f98ed160d24871f1bfd48f3ad68

**Review by SEUNAGBEYE — COMMENTED** [review]
This looks good to me. Get others to review and you need to rebase to fix your merge conflicts.

**Review by Mlg29 — COMMENTED** [review]
-Nice job, but the select element text suppose to be "See all Cities" and not select all cities
- Secondly, increase the height of the appstore and google store
-Thirdly, Terms and conditions and privacy policy isn't included in your footer
-Lastly, try to make it responsive

**Review by Joggz — COMMENTED** [review]
Look Good to me. it's a good one but you are missing the last part
- The terms and condition and the copyright part
lario1

**Review by Joggz — APPROVED** [review]
It is Good to Go.

**Review by Mlg29 — APPROVED** [review]
LGTM

File: dae-alright/src/components/FormInput/FormInput.component.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```
Comment (SEUNAGBEYE) [inline dae-alright/src/components/FormInput/FormInput.component.jsx]: delete this file

Comment (SEUNAGBEYE) [reply dae-alright/src/components/FormInput/FormInput.component.jsx]: and `formInput.style.csss`

**Review by SEUNAGBEYE — CHANGES_REQUESTED** [review]
A few changes requested and this is good to go

**Comment by SEUNAGBEYE** [comment]
Don't forget to rebase too.

File: dae-alright/src/components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import './Footer.scss'
+
+
+function Footer() {
+    return (
+        <footer className="deyAlrightFooterContainer">
+            <div className="logoDiv">
+             <img src="https://res.cloudinary.com/dq9utkjw2/image/upload/v1572357304/logo_zrlxud.png" alt="company-logo" className="footerLogo" />
```
Comment (LuthfulahiO) [inline dae-alright/src/components/Footer/Footer.js]: Which company's logo? Alt should be descriptive, that is what the screen reader wants to read to a visually impaired person. Kindly update your alt property appropriately.

File: dae-alright/src/components/Footer/Footer.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                    <div className="faqAndContactUs">
+                        <ul className="faqBlog">
+                              <li><a href="footer.js">FAQs</a></li>
+                             <li><a href="footer.js">Blog</a></li>
+                        </ul>
+                        <ul className="contactUs">
+                         <li><a href="footer.js">Contact us</a></li>
+                        </ul>
+                    </div>
+                    <div className="socialImages">
```
Comment (LuthfulahiO) [inline dae-alright/src/components/Footer/Footer.js]: image
These guys are stretched, kindly fix.

File: dae-alright/src/components/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+                        <img src="https://res.cloudinary.com/dtbjhs8a6/image/upload/v1571821215/my%20project/et9bmbe3mjhylwbfmree.png" alt="" />
+                        <img src="https://res.cloudinary.com/dtbjhs8a6/image/upload/v1571821216/my%20project/gix7baujoc1bqeczruru.png" alt="" />
```
Comment (LuthfulahiO) [inline dae-alright/src/components/Footer/Footer.js]: Please add an alt attribute to all images

File: dae-alright/src/components/Footer/Footer.js (hunk 3 above)
Comment (Joggz) [reply dae-alright/src/components/Footer/Footer.js]: changed the alt attribute from company-logo to Dae-Alright Logo

File: dae-alright/src/components/Footer/Footer.js [hunk 6]
```diff
@@ -0,0 +1,61 @@
+import React from 'react'
```
Comment (SEUNAGBEYE) [inline dae-alright/src/components/Footer/Footer.js]: rename this file to `index.js`

File: dae-alright/src/App.js [hunk 7]
```diff
@@ -1,14 +1,16 @@
...
 
 
 class App extends Component {
 
   render(){
   return (
     <div className="App">
-      <h1>dae-Alright</h1>
+      
+
```
Comment (SEUNAGBEYE) [inline dae-alright/src/App.js:15]: Use the footer here

File: dae-alright/src/App.js [hunk 8]
```diff
@@ -1,14 +1,16 @@
 import React, { Component } from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './App.css';
+import Footer from './components/Footer/Footer'
```
Comment (SEUNAGBEYE) [inline dae-alright/src/App.js]: import as `import Footer from './components/Footer`

**Review by SEUNAGBEYE — CHANGES_REQUESTED** [review]

**Review by SEUNAGBEYE — APPROVED** [review]
LGTM

**Pull request merged**